collect_app/src/main/java/org/odk/collect/android/formlists/blankformlist/BlankFormListViewModel.kt

```diff
@@ -75,14 +79,29 @@ class BlankFormListViewModel(
 
     fun syncWithServer(): LiveData<Boolean> {
         val result = MutableLiveData<Boolean>()
-        scheduler.immediate(
-            {
-                formsDataService.matchFormsWithServer(projectId)
-            },
-            { value: Boolean ->
-                result.value = value
-            }
-        )
+
+        if (FeatureFlags.FOREGROUND_SERVICE_UPDATES) {
+            scheduler.immediate(
```

Comment:
With the new approach, there is no visual confirmation if the sync finished successfully or not. There should be something, I think.